### ISSUE TITLE
Add support for (amsmath) multlined environment

### ISSUE THREAD

**Issue by wyleyr** [jgm/texmath#210]
I was pleased to discover that texmath and pandoc already support most of the amsmath environments! Unfortunately I am trying to convert a document that makes use of the multlined environment (the version of multline that can be used inside another equation environment) and pandoc doesn't recognize it, so this is a feature request to add support for this environment.

I hope this is as simple as  adding
```haskell
  , ("multlined", gather)
```
to the list of environments in src/Text/TeXMath/Readers/TeX.hs but I'm no expert on how this code works.

**Comment by jgm** [comment]
Yes, I think it is that simple. Note: as tex interprets multiline, the first part will be aligned to the left and the second part will be displayed in the next line and aligned to the right.  texmath won't capture this formatting detail.

**Comment by wyleyr** [comment]
Thanks John! I am always astounded by your response times.

**Comment by jgm** [comment]
Hi!  I didn't recognize your handle.  I've found that either I do it right away or it languishes for years...
